usig/sgx/usig-enclave.go

```diff
@@ -16,12 +16,66 @@
 
 package sgx
 
-// #cgo pkg-config: $SGX_URTS_PACKAGE
-// #cgo CFLAGS: -I${SRCDIR}/shim
-// #cgo LDFLAGS: -L${SRCDIR}/shim -lusig_shim
-//
-// #include <stdlib.h>
-// #include "usig.h"
+/*
+// We use only a value of `Cflags` from pkg-config.
+// So, this works also for the hardware mode.
+#cgo pkg-config: libsgx_urts_sim
```

Comment:
Conversely, why don't use `libsgx_urts`?

Reply:
I don't mind `_sim` or not. I will change to more fundamental one `libsgx_urts`.